underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -159,6 +165,26 @@ public SwiftUnderFileSystem(AlluxioURI uri) {
       container.create();
     }
     mContainerPrefix = Constants.HEADER_SWIFT + mContainerName + PATH_SEPARATOR;
+
+    // Assume the Swift user name has 1-1 mapping to Alluxio username.
+    mAccountOwner = Configuration.get(PropertyKey.SWIFT_USER_KEY);
+    short mode = (short) 0;
+    // If there is any container ACL for the Swift user, translates it to Alluxio permission.
+    if (container.getContainerReadPermission().contains(mAccountOwner)
```

Comment:
Thanks @calvinjia , I've added the documentation. What do you mean by configurable strictness? For now we only have one way to check ACL, which is pretty loose. Maybe we can add strictness config later once we had more options?